src/dBFT/DBFTPlugin.cs

```diff
@@ -16,10 +17,23 @@ protected override void Configure()
             Settings.Load(GetConfiguration());
         }
 
+        protected override void OnPluginsLoaded()
+        {
+            walletProvider = GetService<IWalletProvider>();
+            if (Settings.Default.AutoStart)
+                walletProvider.WalletOpened += WalletProvider_WalletOpened;
+        }
+
+        private void WalletProvider_WalletOpened(object sender, Wallet wallet)
```

Comment:
What will be the message is wallet closed?

Reply:
It will pass the wallet with `null` value.